package.json

```diff
@@ -79,7 +79,7 @@
         "karma-mocha-reporter": "2.2.5",
         "karma-sourcemap-loader": "0.3.7",
         "karma-spec-reporter": "0.0.32",
-        "karma-webpack": "3.0.0",
+        "karma-webpack": "4.0.0-rc.2",
```

Comment:
rc -  means release candidate?

Reply:
This was incorrect. I've rolled back this version change. It can be revisited once the final is released.